app/src/components/new-request/stats.vue

```diff
@@ -125,22 +136,43 @@ export default class Stats extends Vue {
 
   /** The regular wait time, in days. */
   get regularWaitTime (): string | number {
-    const regularWaitTime = GetFeatureFlag('hardcoded_regular_wait_time')
-    if (regularWaitTime > 0) {
-      return regularWaitTime
-    } else {
-      return (this.getStats?.regular_wait_time ?? '-')
+    const flagVal = GetFeatureFlag('hardcoded_regular_wait_time')
+
+    if (flagVal > 0) return flagVal
+    if (flagVal < 0) return '-'
+
+    // when flag is 0 or invalid, fall back to stats value
+    const statVal = this.getStats?.regular_wait_time
+    if (statVal > 0) {
+      return statVal
     }
+
+    return '-'
   }
 
-  /** The priority wait time, in hours. */
+  /** The priority wait time, in days. */
   get priorityWaitTime (): string | number {
-    const priorityWaitTime = GetFeatureFlag('hardcoded_priority_wait_time')
-    if (priorityWaitTime > 0) {
-      return priorityWaitTime
-    } else {
-      return (this.getStats?.priority_wait_time ?? '-')
+    const flagVal = GetFeatureFlag('hardcoded_priority_wait_time')
+
+    if (flagVal > 0) return flagVal
+    if (flagVal < 0) return '-'
+
+    const statVal = this.getStats?.priority_wait_time
+    if (statVal > 0) {
+      return statVal
     }
+
+    return '-'
+  }
+
+  get showRegularTooltip (): boolean {
+    const val = Number(this.regularWaitTime)
+    return Number.isFinite(val) && val > 0
```

Comment:
I think you don't need all the checks here... This works:

<img width="318" height="219" alt="image" src="https://github.com/user-attachments/assets/b86d47cb-c745-499f-a9fa-e9c930c89a14" />

But, if you want to keep this, I'm OK with it.